src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -116,6 +120,11 @@ object Coroutine {
       frame
     }
 
+    /** Advances the coroutine to the next yielding point.
```

Comment:
Did you push your changes? Seems like the comments were not addressed in the PR..